Makefile

```diff
@@ -16,6 +16,11 @@
 IMPORT_PATH=github.com/intel/pmem-csi
 SHELL=bash
 
+#FIXME: replace --all with --tags once we started release tagging
+ifeq ($(VERSION), )
+VERSION=$(shell git describe --long --dirty --all)
```

Comment:
oim example used --tags --match=v* to tie versioning to tag lines, we don't want/plan that?
It would require to give first v-form tag in our repo to become functional

Reply:
> oim example used --tags --match=v* to tie versioning to tag lines, we don't want/plan that?
> It would require to give first v-form tag in our repo to become functional

I  mentioned this in #FIMXE comment. Once we are ready with release tagging we can move to  `-tags --match=v*` in place of `--all`.

Reply:
ah yes, I was too quick and mislooked FIXME. What about tagging now with some baseline like v0.0.0 so that tag option will always work (I think OIM did similar?). Then we don't need to maintain that FIXME place in code